src/opm/output/eclipse/AggregateUDQData.cpp

```diff
@@ -727,7 +732,7 @@ const std::vector<int> Opm::RestartIO::Helpers::igphData::ig_phase(const Opm::Sc
                     phase_sum += 2;
                 if (group.hasInjectionControl(Opm::Phase::GAS))
                     phase_sum += 4;
-                inj_phase[group.insert_index()] = phase_sum;
+                inj_phase[group.insert_index()-1] = phase_sum;
```

Comment:
The `curGroups` variable will contain all groups - including the `FIELD` group which has `insert_index == 0` - so the `[group.insert_index() - 1]` looks dangerous.

Reply:
When here I guess the comment on lines 723 - 727 can be removed.

Reply:
Thanks , yes - the index here will be wrong for the "FIELD" group! - code is / will be corrected to handle this case.